presenters/thread_list.rb

```diff
@@ -6,11 +6,13 @@ class ThreadListPresenter
   def initialize(threads, user, course_id)
     read_states = ThreadUtils.get_read_states(threads, user, course_id)
     threads_endorsed = ThreadUtils.get_endorsed(threads)
+    threads_flagged = ThreadUtils.get_abuse_flagged_count(threads)
```

Comment:
If this is an expensive operation, we should probably only perform it if an abuse count is explicitly requested, if a user is a moderator.